client_test/stub_test.py

```diff
@@ -164,48 +164,6 @@ def test_same_function_name(caplog):
     assert "square" in caplog.text
 
 
-skip_in_github = pytest.mark.skipif(
```

Comment:
don't think it was worth keeping these tests – `stub.serve` was kind of a dummy shim anyway, and these tests are skipped in github.

We have tests for live reloading anyway.

Reply:
:+1: